src/form-view.js

```diff
@@ -141,7 +146,14 @@ const viewMethods = {
         icon.pack,
         `${icon.prefix}${icon.name}${icon.suffix}`
       );
-      iconView.textContent = icon.description;
+
+      if (iconsListStyle) {
+        iconView.textContent = "";
+        this.disposables.add(atom.tooltips.add(iconView, {title: icon.description}));
```

Comment:
there is a property in the object options that you can change the timeouts on show and hide, `{title: icon.description, delay: {show: 100, hide: 100}}`